action.yml

```
@@ -21,6 +21,8 @@ inputs:
     default: ${{ github.token }}
   cache:
     description: 'Used to specify a package manager for caching in the default directory. Supported values: npm, yarn, pnpm'
+  cache-dependency-path:
+    description: 'Used to specify path to a dependencies file: package-lock.json, yarn.lock, etc. Supports wildcards or an array of file names.'
```

```suggestion
    description: 'Used to specify the path to a dependency file: package-lock.json, yarn.lock, etc. Supports wildcards or a list of file names for caching multiple dependencies.'
```

`List` makes more sense vs `array` since you can't do something like

```
    cache-dependency-path: |
      [server/app/package-lock.json
      frontend/app/package-lock.json]
```